data/beps/beps.yml

```
@@ -600,14 +601,15 @@
     -   given-names: Rémi
         family-names: Gau
     status:
-    -   To harmonize and make more reusable stimuli content under stimuli/.
+    -   Community comments and feedback are being collected (January 2025)
+    -   To harmonize and make more reusable stimuli content under stimuli/
     -   Collecting community comments and feedback. All collaborators are welcome.
     -   'Original issue: [#153](https://github.com/bids-standard/bids-specification/issues/153)'
     content:
     -   raw
     blocking:
-    google_doc_created: 2023-09-12
-    pull_request_created:
+    google_doc_created: 2023-09
+    pull_request_created: 2024-12
```

I would prefer to stick to the already used format since code might rely on it:

```suggestion
    google_doc_created: 2023-09-12
    pull_request_created: 2024-12-21
```